google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java

```diff
@@ -273,6 +302,7 @@ SessionImpl createMultiplexedSession() {
               spanner,
```

Comment:
GitHub does not allow me to comment on the actual line, but the call to `getRpc().createSession(..)` above should also be passed a request id.

Reply:
I've made the change, thanks @olavloite!